cmd/bee/cmd/start.go

```diff
@@ -346,6 +347,22 @@ func buildBeeNode(ctx context.Context, c *command, cmd *cobra.Command, logger lo
 
 	return b, err
 }
+func (c *command) CheckMissingFlag(cmd *cobra.Command, args []string) error {
```

Comment:
This should be named `CheckUnknownParams` and it should be moved to the `cmd.go` file.